src/Forum/Console/EnableUserPassAuthCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Forum\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Settings\SettingsRepositoryInterface;
+
+class EnableUserPassAuthCommand extends AbstractCommand
```

Comment:
Oh and I forgot one thing.

I don't think it makes sense to create this command. We might as well have a single command to edit any setting, and explain in the documentation how to re-enable password login if necessary.

Reply:
Removed, I agree that'd be a better approach (although probably best for a separate PR). Were you envisioning that users input json encoded strings directly?